app/src/main/java/org/wikiedufoundation/wikiedudashboard/ui/profile/view/ProfileStatsFragment.kt

```diff
@@ -58,32 +59,32 @@ class ProfileStatsFragment : Fragment() {
         profileResponse?.asStudent?.let {
             val asStudentDetails: AsStudentDetails = it
             val text = "Total impact made by $username's students"
-            tvTitleImpactByStudent.text = text
-            tvCountArticlesCreated.text = asStudentDetails.individualArticlesCreated
-            tvCountArticlesEdited.text = asStudentDetails.individualArticleViews
-            tvCountTotalEdits.text = asStudentDetails.individualArticleCount
-            tvCountStudentEditors.text = asStudentDetails.individualUploadCount
-            tvCountWordsAdded.text = asStudentDetails.individualWordCount
-            tvCountArticleViews.text = asStudentDetails.individualArticleViews
-            tvCountCommonsUploads.text = asStudentDetails.individualUploadCount
+            tv_title_impact_by_student.text = text
+            tv_count_articles_created.text = asStudentDetails.individualArticlesCreated
+            tv_count_articles_edited.text = asStudentDetails.individualArticleViews
+            tv_count_total_edits.text = asStudentDetails.individualArticleCount
+            tv_count_student_editors.text = asStudentDetails.individualUploadCount
+            tv_count_words_added.text = asStudentDetails.individualWordCount
+            tv_count_article_views.text = asStudentDetails.individualArticleViews
+            tv_count_commons_uploads.text = asStudentDetails.individualUploadCount
         } ?: run {
-            llAsStudent.visibility = GONE
-            llNotEnrolled.visibility = VISIBLE
+            cl_as_student.visibility = GONE
+            tv_not_enrolled.visibility = VISIBLE
```

Comment:
Change Naming Convention from 'snake_case' to 'camelCase' in all elements.

Reply:
I have changed the naming convention from snake_case to camelCase